image/templates/helm/shared/templates/02-scanner-v4-07-indexer-deployment.yaml

```diff
@@ -157,7 +157,16 @@ spec:
       - name: etc-pki-volume
         emptyDir: {}
       {{- if ._rox._securedClusterCertRefresh }}
-      {{- include "srox.tlsCertsInitContainerVolumes" (list "scanner-v4-indexer") | indent 6 }}
+      - name: certs
+        emptyDir: { }
```

Comment:
```suggestion
        emptyDir: {}
```

Reply:
Thanks! I copied the code from git history to make sure it is exactly as it was, and then the IDE proceeded to change it. 😠